src/index.html

```diff
@@ -5,8 +5,415 @@
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <!-- linthtml-configure attr-req-value="false" -->
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link
+    href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+    rel="stylesheet"
+    crossorigin
+    >
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+
+      <a class= "container" href="#">
+        <img
+        src="https://raw.githubusercontent.com/mmahfuzi/layout_moyo-header-en/master/src/images/logo.png"
+        alt="logo">
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a class="nav__link is-active" href="#">Apple</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#samsung">Samsung</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#smartphones">Smartphones</a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              data-qa="nav-hover"
+              href="#laptops & computers"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#gadgets">Gadgets</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#tablets">Tablets</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#photo">Photo</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#video">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="cards">
+      <main class="cards-container cards__centered">
+        <section class="cards-container__page page">
+          <div class="card-block">
+            <article data-qa="card" class="card card-block__distance">
+              <img
+                class="card__image image"
+                src="./images/imac.jpeg"
+                alt="Laptop"
+              >
+
+              <div class="card__text">
+                <p class="card__text card__text--1">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</p>
+                <p class="card__text card__text--2">Product code: 195434</p>
+              </div>
+
+              <div class="card__rate">
+                <div class="stars stars--4">
+                  <div class="stars__star"></div>
+                  <div class="stars__star"></div>
+                  <div class="stars__star"></div>
+                  <div class="stars__star"></div>
+                  <div class="stars__star"></div>
+                </div>
+
+                <div class="card__rate-review">Reviews: 5</div>
+              </div>
+
+              <p class="card__price">
```

Comment:
<img width="484" alt="Monosnap Catalog 2022-11-10 16-40-06" src="https://user-images.githubusercontent.com/39308717/201120792-b9752e0d-786d-46b1-940c-2bf30fce39eb.png">

Reply:
this one is still there @mmahfuzi 
this is the last one 🙏